.jenkins/build.sh

```diff
@@ -151,6 +151,14 @@ elif [[ "${JOB_TYPE}" == "manager" ]]; then
   # Step 7: push new HTML files and static files to gh-pages
   if [[ "$COMMIT_SOURCE" == "refs/heads/master" || "$COMMIT_SOURCE" == "refs/heads/main" ]]; then
     git clone https://github.com/pytorch/tutorials.git -b gh-pages gh-pages
+    # Clean up directories that contain tutorials
+    
+    for dir in beginner intermediate prototype recipes advanced distributed vision text audio; do
+      if [ -d "gh-pages/$dir" ]; then
+        rm -rf "gh-pages/$dir"
+      fi
```

Comment:
Technically one does not need `-d` as `rm -rf` returns 0 even if folder does not exist
```
% ls kjhkjhkjh; echo $?
ls: kjhkjhkjh: No such file or directory
1
% rm -rf kjhkjhkjh; echo $?
0
```
```suggestion
      rm -rf "gh-pages/$dir"
```